ui/css/module/_styleguide.scss

```diff
@@ -1,26 +1,22 @@
+$basic-spacing: 1rem;
+$medium-spacing: 2.5rem;
```

Comment:
Probably better to use the [existing](https://github.com/18F/omb-eregs/blob/9b0b4e8302662b2dcb729ec7f549281b18f5c0bb/ui/css/base/_spacing.scss) spacing vars, yes?